k8s/service_controller/model/service_broker.go

```diff
@@ -0,0 +1,11 @@
+package model
+
+type ServiceBroker struct {
+	// json info from create docs
+	// http://apidocs.cloudfoundry.org/239/service_brokers/create_a_service_broker.html
+	Name         string `json:"name"`
+	BrokerURL    string `json:"broker_url"`
```

Comment:
is there going to be any negotiation on this? It seems overly verbose to call something 'broker_url' in a 'broker' object.